Xamarin.Forms.Core/IndicatorView.cs

```diff
@@ -106,18 +106,22 @@ public IEnumerable ItemsSource
 			set => SetValue(ItemsSourceProperty, value);
 		}
 
-		public static readonly BindableProperty ItemsSourceByProperty = BindableProperty.Create("ItemsSourceBy", typeof(VisualElement), typeof(IndicatorView), default(VisualElement), propertyChanged: (bindable, oldValue, newValue)
-		 => LinkToCarouselView(bindable as IndicatorView, newValue as CarouselView));
 
 		[TypeConverter(typeof(ReferenceTypeConverter))]
-		public static VisualElement GetItemsSourceBy(BindableObject bindable)
+		public VisualElement ItemsSourceBy
 		{
-			return (VisualElement)bindable.GetValue(ItemsSourceByProperty);
+			get => (VisualElement)GetValue(ItemsSourceByProperty);
+			set => SetValue(ItemsSourceByProperty, value);
 		}
 
-		public static void SetItemsSourceBy(BindableObject bindable, VisualElement value)
+		public static readonly BindableProperty ItemsSourceByProperty = BindableProperty.Create(nameof(ItemsSourceBy), typeof(VisualElement), typeof(IndicatorView), default(VisualElement), propertyChanged: (bindable, oldValue, newValue)
+		 => (bindable as IndicatorView)?.LinkToVisualElement(newValue as VisualElement));
+
+
+		public virtual void LinkToVisualElement(VisualElement visualElement)
 		{
-			bindable.SetValue(ItemsSourceByProperty, value);
+			if (visualElement is CarouselView carouselView)
```

Comment:
Can we link to an ItemsView by default, maybe?

Reply:
Sure